errbot/backends/irc.py

```diff
@@ -254,7 +267,7 @@ def occupants(self):
         occupants = []
         try:
             for nick in self._bot.conn.channels[self.room].users():
-                occupants.append(IRCMUCOccupant(nick=nick, room=self.room))
+                occupants.append(IRCMUCOccupant(nick, self.room))
```

Comment:
Ah, indeed here is not nick the kwarg name, is mask, that's why i remove both, to avoid this kind of issues.